### ISSUE TITLE
indexlr fails to read the barcode (10x format)

### ISSUE THREAD

**Issue by aafshinfard** [bcgsc/btllib#73]
Should be an easy fix:

I have a toy example:
```
$ cat data/stLFR2.tiny.fq
@V10002828L1C001R013000000#543_288_92/1 1       1
TCTTCAACAGAA
+
JJJJJJJJJJJJ
```
and when I try indexlr it fails to extract the barcode from the header (reports NA)
```
$ indexlr --bx -t16 -k8 -w1 data/stLFR2.tiny.fq
indexlr 1.4.9: Using more than 5 threads does not scale, reverting to 5.
NA      14450190702609987575 11048730314773898660 4535684329547842690 9637752013492521276 8090755262123482019
```
but I see the logic is there for this type of header.
https://github.com/bcgsc/btllib/blob/8919e1ef317bb9b61699b2f01d78c9d81524b236/include/btllib/indexlr.hpp#L400

(side note: I believe I installed btllib 1.4.10 but indexlr says 1.4.9, so maybe version naming inconsistency also?)

**Comment by aafshinfard** [comment]
Should be an easy fix:

I have a toy example:
```
$ cat data/stLFR2.tiny.fq
@V10002828L1C001R013000000#543_288_92/1 1       1
TCTTCAACAGAA
+
JJJJJJJJJJJJ
```
and when I try indexlr it fails to extract the barcode from the header (reports NA)
```
$ indexlr --bx -t16 -k8 -w1 data/stLFR2.tiny.fq
indexlr 1.4.9: Using more than 5 threads does not scale, reverting to 5.
NA      14450190702609987575 11048730314773898660 4535684329547842690 9637752013492521276 8090755262123482019
```
but I see the logic is there for this type of header.
https://github.com/bcgsc/btllib/blob/8919e1ef317bb9b61699b2f01d78c9d81524b236/include/btllib/indexlr.hpp#L400

(side note: I believe I installed btllib 1.4.10 but indexlr says 1.4.9, so maybe version naming inconsistency also?)

**Comment by vlad0x00** [comment]
Version name comes from `meson.build` file (https://github.com/bcgsc/btllib/blob/master/meson.build). I forgot to update it for 1.4.10

**Comment by lcoombe** [comment]
This was fixed in #74 - Will be in the next btllib release!
